frontend/src/components/ApprovedReportSpecialButtons.js

```diff
@@ -9,6 +9,8 @@ export default function ApprovedReportSpecialButtons({
   modalRef,
   user,
   showUnlockReports,
+  showCompleteEvent,
+  completeEvent,
```

Comment:
Kinda nit-picky, but I think usually handlers like this have the `on` prefix, like `onCompleteEvent` so it's clear what the prop is since we don't have helpful hover definitions for props like this